client/src/_shared/LoggedInLayout.js

```diff
@@ -12,7 +13,8 @@ const { useBreakpoint } = Grid
 
 export function LoggedInLayout({ children, title }) {
   const dispatch = useDispatch()
-  const { t } = useTranslation()
+  const { t, i18n } = useTranslation()
+  const [language, setLanguage] = useState(i18n.language)
```

Comment:
Why do we need to hold the language state? Can't we use the state available from `react-i18next`?